fusor-ember-cli/app/views/application.js

```diff
@@ -0,0 +1,13 @@
+export default Ember.View.reopen({
+  init() {
+    this._super();
+    var self = this;
+
+    // bind attributes beginning with 'data-'
+    Ember.keys(this).forEach(function(key) {
+      if (key.substr(0, 5) === 'data-') {
+        self.get('attributeBindings').pushObject(key);
+      }
```

Comment:
@cfchase, @eriknelson, this code snippet is from the guides and what allows the data attributes not to be stripped.
https://guides.emberjs.com/v1.13.0/templates/binding-element-attributes/